main.go

```diff
@@ -86,15 +121,23 @@ func main() {
 	}
 	// +kubebuilder:scaffold:builder
 
+	// Next, add a Healthz checker to the manager. Healthz is a health and liveness package
+	// that the operator will use to periodically check the health of its pods, etc.
 	if err := mgr.AddHealthzCheck("health", healthz.Ping); err != nil {
 		setupLog.Error(err, "unable to set up health check")
 		os.Exit(1)
 	}
+
+	// Now add a ReadyZ checker to the manager as well. It is important to ensure that the
+	// API server's readiness is checked when the operator is installed and running.
 	if err := mgr.AddReadyzCheck("check", healthz.Ping); err != nil {
 		setupLog.Error(err, "unable to set up ready check")
 		os.Exit(1)
 	}
 
+	// Now that the manager and controller are all setup, it's time to start the manager.
+	// TheSetupSignalHandler registers for SIGINT and SIGTERM, which can be used to
+	// terminate the manager if they are both called.
```

Comment:
Suggestion: we could be a little less verbose for faster readability

```suggestion
Register signal handler for SIGINT and SIGTERM to terminate the manager
```

Reply:
Replaced my comment with your suggested change